### ISSUE TITLE
`queryInfo` and `queryFeeDetails` of `transactionPaymentApi` Runtime API are broken in the developer tab

### ISSUE THREAD

**Issue by IkerAlus** [polkadot-js/apps#10994]
When querying any of the `queryInfo` and `queryFeeDetails` methods of the `transactionPaymentApi` Runtime API in any chain from the developer tab the following error is returned:

```
4003: Client error: Execution failed: Execution aborted due to trap: wasm trap: wasm `unreachable` instruction executed WASM backtrace: error while executing at wasm backtrace: 0: 0x9be8 - <unknown>!rust_begin_unwind 1: 0x319c - <unknown>!core::panicking::panic_fmt::hbb5a6b42001bdfec 2: 0x5730f1 - <unknown>!TransactionPaymentApi_query_fee_details
```

It seems that the [new release of PJS API](https://github.com/polkadot-js/api/releases/tag/v14.0.1) has introduced some changes in certain expected types for these queries on PJS apps.

We had a similar issue on API Sidecar and [converting the extrinsic to u8a](https://github.com/paritytech/substrate-api-sidecar/pull/1505/files) solved the issue.

  - [ X] Bug report
  - [ ] Feature request
  - [ ] Support request
  - [ ] Other

* **Please tell us about your environment:**

  - Environment:

    - [ ] Node.js
    - [x ] Browser
    - [ ] Other (limited support for other environments)

**Comment by TarikGul** [comment]
Yes, this is certainly an odd issue - I would of imagined the api underneath would accept any value and transform it as necessary but this seems to not be the case. This issue is probably related to: https://github.com/polkadot-js/api/releases/tag/v13.2.1

**Comment by Imod7** [comment]
The issue seems to appear when upgrading the `api` from [12.4.2](https://github.com/polkadot-js/api/releases/tag/v12.4.2) to [13.0.1](https://github.com/polkadot-js/api/releases/tag/v13.0.1).

In `v12.4.2` if I call 
```
await apiAt.call.transactionPaymentApi.queryInfo(extrinsic, extrinsic.length);
```
where extrinsic is a Hex string, it will return the fees with no issues.

As soon as I upgrade to `v13.0.1` the same call returns a wasm panic error and I need to change it to 
```
await apiAt.call.transactionPaymentApi.queryInfo(u8a, u8a.length);
```
where u8a is 
```
 const ext = api.registry.createType('Extrinsic', extrinsic);
 const u8a = ext.toU8a();
```
to get the fees with no issues again.


What I noticed in apps is that for the same extrinsic, when it goes in [OnSubmit](https://github.com/polkadot-js/apps/blob/fde0f724f5d1a404ecc5143bcc5cc1e912f98386/packages/page-runtime/src/Runtime/index.tsx#L28) and I check the value of the extrinsic submitted (`values[0].value`) it looks like apps converted the input extrinsic into `Bytes` type 
```
api.registry.createType('Bytes', transaction);
```
and not into `Extrinsic` type like I did above. Maybe this is related to the error ?

The [script](https://gist.github.com/Imod7/026e4ba4872098f53fa16cae6e099cdb) that I used to debug the different versions of the api.
